reana/reana_dev/wiki.py

```diff
@@ -0,0 +1,148 @@
+    """Wiki commands group."""
+
+
+@wiki_commands.command(name="wiki-create-build-status-page")
+def create_build_status_page():
+    """Generate Markdown for the Build Status wiki page."""
+    sections = {
+        "researchers": {
+            "title": "For researchers",
+            "description": "Find out how you can use REANA to describe, run, preserve and reuse your analyses.",
+            "packages": {
+                "reana-client": {},
+                "docs.reana.io": {"coverage": False, "docs": False},
+                "www.reana.io": {"coverage": False, "docs": False},
+            },
+        },
+        "administrators": {
+            "title": "For administrators",
+            "description": "Install and manage the REANA reusable analysis platform on your own compute cloud.",
+            "packages": {
+                "reana-commons": {},
+                "reana-db": {},
+                "reana-job-controller": {},
+                "reana-message-broker": {},
+                "reana-server": {},
+                "reana-ui": {},
+                "reana-workflow-controller": {},
+                "reana-workflow-engine-cwl": {},
+                "reana-workflow-engine-serial": {},
+                "reana-workflow-engine-yadage": {},
+                "reana-workflow-monitor": {},
+            },
+        },
+        "developers": {
+            "title": "For developers",
+            "description": "Understand REANA source code, adapt it to your needs, contribute changes back.",
+            "packages": {"reana": {}, "pytest-reana": {}},
+        },
+        "environments": {
+            "simple": True,
+            "title": "Environments",
+            "description": "Selected containerised environments.",
+            "packages": {
+                "reana-env-aliphysics": {},
+                "reana-env-jupyter": {},
+                "reana-env-root6": {},
+                "reana-auth-krb5": {},
+                "reana-auth-vomsproxy": {},
+            },
+        },
+        "examples": {
+            "simple": True,
+            "title": "Examples",
+            "description": "Selected reusable analysis examples.",
+            "packages": {
+                "reana-demo-alice-lego-train-test-run": {},
+                "reana-demo-alice-pt-analysis": {},
+                "reana-demo-atlas-recast": {},
+                "reana-demo-bsm-search": {},
+                "reana-demo-cdci-crab-pulsar-integral-verification": {},
+                "reana-demo-cdci-integral-data-reduction": {},
+                "reana-demo-cms-h4l": {},
+                "reana-demo-cms-reco": {},
+                "reana-demo-cms-dimuon-mass-spectrum": {},
+                "reana-demo-fcchh-fullsim": {},
+                "reana-demo-helloworld": {},
+                "reana-demo-lhcb-d2pimumu": {},
+                "reana-demo-lhcb-mc-production": {},
+                "reana-demo-root6-roofit": {},
+                "reana-demo-worldpopulation": {},
```

Comment:
It would be great if for all these lists we use the same config that `reana-dev` uses in general, e.g. in this case [`REPO_LIST_DEMO`](https://github.com/reanahub/reana/blob/b962864f6fe23a2864053b7d047aa1ad33e6572a/reana/config.py#L11). CC @tiborsimko is there any reason why `REPO_LIST_DEMO` doesn't contain all demos and part of them are just in `REPO_LIST_ALL`? Maybe with this PR comes the time to harmonise it.

Reply:
`REPO_LIST_DEMO` contains only known "runnable examples", i.e. ones that are fast to run locally on a typical developer box.  They are used for `reana-dev run-ci -c DEMO`.

Hence the other examples, that are "non-runnable" locally, are living outside only as part of `REPO_LIST_ALL`. 

If we want we could amend this and introduce `REPO_LIST_DEMO_RUNNABLE`, `REPO_LIST_DEMO_ALL` for example.

Reply:
Updated.